istio-proxy/istio-proxy.spec

```diff
@@ -21,7 +23,7 @@
 
 Name:           istio-proxy
 Version:        0.1.0
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
Should this be version 0.1.1 and release 1 instead?

Reply:
Good question. I'm not sure.

Reply:
It's a new Istio release that we're picking up. It just happens to be bug fixes.